mill

```diff
@@ -3,6 +3,7 @@
 # Adapted from
 
 coursier_version="2.1.24"
+COMMAND=$@
```

Comment:
This is a mystery, on BASH it should always be the program arguments which on fish it's last invoked command's one. When running bash script from fish it seems to be using fish semantics or maybe it's because cs is being invoked in fish and it's changing the variable? 😓